spinalcordtoolbox/image.py

```diff
@@ -1398,20 +1398,20 @@ def concat_warp2d(fname_list, fname_warp3d, fname_dest):
     warp3d = np.zeros([nx, ny, nz, 1, 3])
 
     for iz, fname in enumerate(fname_list):
-        img = nib.load(fname)
-        warp2d = np.asanyarray(img.dataobj)
+        img = Image(fname)
+        warp2d = np.asanyarray(img.data)
         warp3d[:, :, iz, 0, 0] = warp2d[:, :, 0, 0, 0]
         warp3d[:, :, iz, 0, 1] = warp2d[:, :, 0, 0, 1]
         del warp2d
 
     # save new image
-    im_dest = nib.load(fname_dest)
-    affine_dest = im_dest.affine
-    im_warp3d = nib.nifti1.Nifti1Image(warp3d, affine_dest)
+    im_dest = Image(fname_dest)
+    im_warp3d = Image(warp3d)
+    im_warp3d.copy_qform_from_ref(im_dest)
```

Comment:
This change was a little unique, in that it's meant to copy an affine from one image into another image.

The method `copy_qform_from_ref` is pretty misleading -- the actual method copies both the qform and sform. So, perhaps it should instead be called `copy_affine_from_ref`?

Reply:
Yes, that would be a good rename! I'm in favor of it.